src/core-components/Modal/Modal.less

```diff
@@ -0,0 +1,51 @@
+@import './src/assets/styles/variables/colors.less';
+
+.ant-modal-content {
+  background-color: transparent;
+}
+
+.ant-modal-header {
+  padding: 24px 16px 0;
```

Comment:
```padding: 24px 16px 0 0;```
I think it is more correct.

Reply:
Stylelint will through an Error on it, I tried